mock/py/mockbuild/util.py

```diff
@@ -177,6 +177,8 @@ def orphansKill(rootToKill, manual_forced=False):
                     root = os.readlink("/proc/%s/root" % fn)
                     if compare_two_paths_cached(root, rootToKill, path_cache):
                         getLog().warning("Process ID %s still running in chroot. Killing with %s...", fn, killsig)
+                        with open("/proc/%s/cmdline" % fn) as f:
+                            getLog().warning("Process is %s", f.readline())
```

Comment:
Thank you.  Can this end up with any exception that would potentially skip the `os.kill()` call?